### PR TITLE
Fix for issues https://github.com/PUGX/PUGXMultiUserBundle/issues/146

### PR DESCRIPTION
**Pull request by padalec**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "pugx/multi-user-bundle",
+    "name": "silversite/multi-user-bundle",
```
Comment (garak) [inline composer.json]: ? :confused:

Comment (padalec) [inline composer.json]: hmm I don't want this to be in PR.

File: composer.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "pugx/multi-user-bundle",
+    "name": "silversite/pugx-multi-user-bundle",
```
Comment (garak) [inline composer.json]: Sometimes They Come Back...

**Comment by padalec** [comment]
Try now. I removed this commit from PR.

**Comment by garak** [comment]
Thank you!

**Pull request merged**